study.md

```diff
@@ -87,7 +88,19 @@ Create the following movies with the given attributes.
 | 2 | The Core | 5.4 |
 
 ```ruby
-# your answer here
+movie_ids = %w(0 1 2)
+movie_titles = %w(Battlefield Earth, Sharknado, The Core)
+movie_ratings = %w(2.4 3.3. 5.4)
+
+movies = movie_ids.product(movie_titles).movie_ratings
```

Comment:
This will cause an error because you're trying to call `movie_ratings` as if it were a method on `movie_ids.product(movie_titles)`, which is an array.  Furthermore, `Array#product` returns the Cartesian product of two arrays.  What you're looking for is `Array#zip`, like so: `movies = movie_ids.zip(movie_titles, movie_ratings)`.  You do not want to create records with `id`s since the database does it automatically with a primary key set.

```ruby
Movie.create(title: 'Battlefield Earth', rating: 2.4)
Movie.create(title: 'Sharknado', rating: 3.3)
Movie.create(title: 'The Core', rating: 5.4)
```